src/Components/v2/CollectionsHubsHomepageNav.tsx

```diff
@@ -18,7 +18,6 @@ export const CollectionsHubsHomepageNav = track(
     context_page: AnalyticsSchema.PageName.HomePage,
     context_module: AnalyticsSchema.ContextModule.CollectionHubEntryPoint,
     subject: AnalyticsSchema.Subject.FeaturedCategories,
-    action_type: AnalyticsSchema.ActionType.Impression,
```

Comment:
Is this change removing the actual impression event, or just removing metadata and the event is still firing? (It's possible that there's a second file that changed, I'm not seeing it right now with the relay noise though.)

Reply:
Plus one that we can remove these generated files from the PR. I was looking at the PR in Force and I'm wondering why both of these tracking events are needed, both here in Reaction and there in Force?